person.rb

```diff
@@ -31,11 +32,16 @@ def isof_age?
 
     false
   end
+
+  def add_rentals(rental)
+    @rentals.push(rental)
+    rental.person = self
+  end
 end
```

Comment:
- [OPTIONAL] Great job creating a method for adding rentals  👍🏻 but it should look like this:

```ruby
 def add_rental(book, date)
    # *The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
    Rental.new(date, book, self)
  end
```

Here is a [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship`) to the example in a previous lesson: